test/validate/eia_test.py

```diff
@@ -46,17 +46,17 @@ def test_no_null_cols_eia(pudl_out_eia, live_dbs, cols, df_name):
 @pytest.mark.parametrize(
     "df_name,raw_rows,monthly_rows,annual_rows",
     [
-        ("bf_eia923", 1_607_692, 1_607_692, 128_252),
+        ("bf_eia923", 1_626_501, 1_626_501, 134_619),
         ("bga_eia860", 142_391, 142_391, 142_391),
-        ("boil_eia860", 83_418, 83_418, 83_418),
-        ("frc_eia923", 656_388, 268_352, 25_443),
-        ("gen_eia923", None, 5_179_478, 433_336),
-        ("gens_eia860", 557_369, 557_369, 557_369),
-        ("gf_eia923", 2_933_836, 2_933_836, 246_324),
+        ("boil_eia860", 86_675, 86_675, 86_675),
+        ("frc_eia923", 661_976, 270_947, 26_470),
+        ("gen_eia923", None, 5_179_478, 433_327),
```

Comment:
Do we know what's up with `gen_eia923`? Why would it have no change to the monthly rows, and lose 9 annual rows?

Reply:
yeah that was indeed a little weird. I didn't look into it because in the past we have overlooked these tiny row fluctuations. If you think it's worth a poke I will dig more!

Reply:
I was more wondering why it isn't changing much more and in both the yearly and monthly cases, in line with the ~5% difference that's seen in the other tables.

Reply:
I think what we want to pay attention to is how big the deviation is from the expected value. Often we don't expect to see any change, so a tiny change is fine. But when we expect to see a significant change (5% more records) and we see no change or a tiny change, that seems suspicious.

Reply:
Here we don't expect the BGA or ownership tables or emissions equipment tables from EIA-860 to change (and they don't), because they only draw from the full EIA-860 data, but I kind of do expect `gen_eia923` to change, since we're adding several new months of EIA-923 data.

Reply:
I tried digging into this, but I can't seem to get any answers. It might be helpful to pair on this?

Reply:
Looking at the latest dates that show up in the tables which can be returned by `PudlTabl.gen_eia923()`

```py
gen_eia923_ms = pd.read_sql("out_eia923__monthly_generation", pudl_engine)
gen_eia923_ys = pd.read_sql("out_eia923__yearly_generation", pudl_engine)
gf_by_gen_eia923_ms = pd.read_sql("out_eia923__monthly_generation_fuel_by_generator", pudl_engine)
gf_by_gen_eia923_ys = pd.read_sql("out_eia923__yearly_generation_fuel_by_generator", pudl_engine)
frc_eia923 = pd.read_sql("out_eia923__monthly_fuel_receipts_costs", pudl_engine)

print(f"gen MS: {gen_eia923_ms.report_date.max()}")
print(f"gen YS: {gen_eia923_ys.report_date.max()}")
print(f"gen fuel by gen MS: {gf_by_gen_eia923_ms.report_date.max()}")
print(f"gen fuel by gen YS: {gf_by_gen_eia923_ys.report_date.max()}")
print(f"frc MS: {frc_eia923.report_date.max()}")

# gen MS: 2023-12-01 00:00:00
# gen YS: 2022-01-01 00:00:00
# gen fuel by gen MS: 2022-12-01 00:00:00
# gen fuel by gen YS: 2022-01-01 00:00:00
# frc MS: 2023-11-01 00:00:00
```